site/src/modules/templates/TemplateFiles/TemplateFiles.tsx

```diff
@@ -104,7 +106,8 @@ export const TemplateFiles: FC<TemplateFilesProps> = ({
 
                     <div css={{ marginLeft: "auto" }}>
                       <Link
-                        to={`/templates/${templateName}/versions/${versionName}/edit?path=${filename}`}
+                        // TODO: skip org name if we're not licensed
```

Comment:
Should we create a ticket for this? I'm concerned that we forget about this TODO.